staff_graded/staff_graded.py

```diff
@@ -93,6 +103,16 @@ def student_view(self, context=None):
         context['display_name'] = self.display_name
         context['is_staff'] = self.runtime.user_is_staff
 
+        course_id = self.location.course_key
+        context['available_cohorts'] = [cohort.name for cohort in get_course_cohorts(course_id=course_id)]
+        if CourseMode:
+            context['available_tracks'] = [
+                (mode.slug, mode.name) for mode in
+                CourseMode.modes_for_course(course_id, only_selectable=False)
+                ]
```

Comment:
oof, feels like we should set up pylint for this sort of indentation weirdness

Reply:
@davestgermain did you set up pylint in your updates to this branch? Shall I enter a ticket for adding it to this repo?

Reply:
Yes. Pylint had no problem with those lines.